src/main/java/org/semux/gui/panel/SendPanel.java

```diff
@@ -308,7 +308,7 @@ public synchronized void actionPerformed(ActionEvent e) {
     protected void refresh() {
         List<WalletAccount> list = model.getAccounts();
 
-        Set<AccountItem> accountItems = new TreeSet<>();
+        Set<AccountItem> accountItems = new LinkedHashSet<>();
```

Comment:
Why not a plain, old, and boring :) ArrayList? It's a list of accounts, isn't it?

Reply:
Thought we were doing contains() calls below, but it looks like we're not.

Reply:
You can use 'contains' on array lists when there are not to many elements. How many? It depends, but sometimes array list is much faster, because of it's simplicity.

Reply:
Like i said, we weren't. :D so all good.

But, afaik, the wallet supports an unlimited number of addresses.